functions/task3.js

```diff
@@ -0,0 +1,8 @@
+const counter = ( num ) => {
```

Comment:
function name should be an action like `countDown`

Reply:
Updated the names of function.